docs/database/usage/index.md

```diff
@@ -335,3 +335,38 @@ and caching. To disable this functionality, update the `database_persistence_ena
 ```
 
 To enable persistence, view the [Offline Support](/database/offline-support) documentation.
+
+# Change the database URL
```

Comment:
Could you move this section about `# firebase.json`? Also, can the name be "Using a secondary database" instead please.

Reply:
done